build-scripts/npm-client.js

```diff
@@ -80,27 +80,29 @@ switch (pkg.name) {
   case 'google-closure-compiler-linux':
   case 'google-closure-compiler-osx':
   case 'google-closure-compiler-windows':
-    // We only want to publish the linux, osx or windows package from a Travis instance running on the correct os
-    if (pkg.name === 'google-closure-compiler-linux' && process.platform !== 'linux' ||
-        pkg.name === 'google-closure-compiler-osx' && process.platform !== 'darwin' ||
-        pkg.name === 'google-closure-compiler-windows' && process.platform !== 'win32') {
-      logMessageQueue.push(`    skipping publication of ${pkg.name} - wrong platform`);
-      process.exitCode = 0;
-    } else {
-      npmPublish(pkg)
-          .then(() => {
-            logMessageQueue.push('  ✅ publish succeeded');
-            logToFile(logMessageQueue.join('\n'));
-          })
-          .catch(err => {
-            process.exitCode = 1;
-            logMessageQueue.push('   ❌ publish failed');
-            logToFile(logMessageQueue.join('\n'));
-            return Promise.reject(err);
-          });
+    // Github actions can publish all versions from the same run, but Travis and Appveyor can't
+    if (!process.env.GITHUB_ACTIONS) {
```

Comment:
Is there still a world where Travis and Appveyor will be triggered?

Reply:
Only if the PR is merged before those 3rd party services are de-provisioned. I just wanted to be defensive here.

Reply:
Ok, I'm not sure what the turn-around time on this is. Generally I thought it was low enough that we could squash this into a single PR, but you're better aware of the limitations here than I am.

Reply:
When we are ready to merge, I can disconnect those services, change that code block and do one more full run of the build to get all green.

Reply:
I removed this check and added scripts to ensure that any necessary artifacts exist in the package before publishing.